resolwe/flow/serializers/annotations.py

```diff
@@ -80,6 +81,13 @@ class AnnotationsSerializer(serializers.Serializer):
     value = serializers.JSONField()
 
 
+class AnnotationsByPathSerializer(serializers.Serializer):
+    """Serializer that reads annotation field and its value."""
+
+    field_path = serializers.CharField(max_length=2 * ANNOTATION_NAME_LENGTH + 1)
```

Comment:
Add a number where the number comes from.

Reply:
You mean a description? Added.